tasks/rubocop.rake

```diff
@@ -0,0 +1,15 @@
+namespace :rubocop do
+  desc 'Runs rubocop on hearth'
+  task 'hearth' do
+    Dir.chdir('hearth') do
+      sh('bundle exec rubocop -E -S')
```

Comment:
If we want to keep bundle exec in here, I think we should probably do something like 
```
sh('BUNDLE_GEMFILE=../Gemfile bundle exec rubocop -E -S')
```